src/config.json

```diff
@@ -79,6 +79,20 @@
         "sources": ["http://localhost:8080/example/wish-list"]
       }
     },
+    {
+      "queryLocation": "books.rq",
+      "name": "Mixed book query",
```

Comment:
Rather rename to "A book query testing sources with and without authentication required"